test-plone-4.3.x.cfg

```diff
@@ -6,5 +6,9 @@ extends =
 package-name = ftw.collectionblock
 
 [versions]
+# Old Pillow versions can't be built on OSX Catalina with Brews zlib.
+# Pillow = 6.2.0
 # Plone 4 support got dropped in 1.4.0
-collective.z3cform.datagridfield = <1.4.0
\ No newline at end of file
+collective.z3cform.datagridfield = <1.4.0
+# The latest versions of setuptools spits out strange warnings
+setuptools = 40.9.0
```

Comment:
This should be pinned generally in ftw-buildouts - not in the package itself. I assume it will be an issues in other packages too then.